models/odos/odos_trades.sql

```diff
@@ -21,16 +21,9 @@ FROM (
         version,
         block_date,
         block_time,
-        token_bought_symbol,
-        token_sold_symbol,
-        token_pair,
-        token_bought_amount,
-        token_sold_amount,
-        token_bought_amount_raw,
-        token_sold_amount_raw,
         amount_usd,
-        token_bought_address,
-        token_sold_address,
+        tokens_bought,
+        tokens_sold,
```

Comment:
Since tokens_bought/tokens_sold is not defined in the base models, you may need to put columns available in base models here to fix.
https://github.com/duneanalytics/spellbook/actions/runs/5169331430/jobs/9311513716?pr=3310#step:9:22

Reply:
I did add these columns in the base models. I see that the error here says `ERROR creating incremental model test_schema.git_ab46cf33_dex_aggregator_trades `. Could the error be due to the model trying to build incrementally? Incremental load wont work now because I change the underlying base model. I need to somehow run dbt with a full refresh for this model